openstack/content-repo/templates/configmap.yaml

```diff
@@ -36,14 +36,25 @@ data:
     {{end}}
     jobs:
       {{- range $index, $job := $config.jobs}}
-      - from: {{$job.source}}
-        to:   {{$job.target}}
+      - from:      {{$job.from}}
+        to:        {{$job.to}}
         {{- if $job.cert}}
-        cert: {{$job.cert}}
-        key:  {{$job.key}}
+        cert:      {{$job.cert}}
+        key:       {{$job.key}}
         {{- end}}
         {{- if $job.ca}}
-        ca:   {{$job.ca}}
+        ca:        {{$job.ca}}
+        {{- end}}
+        {{- if $job.except}}
+        except:    {{$job.except}}
+        {{- end}}
+        {{- if $job.only}}
+        only:      {{$job.only}}
+        {{- end}}
+        {{- if $job.immutable}}
+        immutable: {{$job.immutable}}
+        {{- else if $config.immutable}}
+        immutable: {{$config.immutable}}
         {{- end}}
```

Comment:
Since you're just passing through the `$config.jobs` into the generated YAML, you can use the `toYaml` function to greatly simplify this. See https://github.com/sapcc/helm-charts/blob/2c0bb02b8c710d2ad3a50e07a5ec0a53d77cb45e/openstack/limes/templates/configmap.yaml#L23-L24